src/test/java/uk/gov/hmcts/ccd/data/casedetails/search/SearchQueryFactoryOperationTest.java

```diff
@@ -31,6 +32,7 @@
     private static final String TEST_FIELD_VALUE = "Tim";
 
     private SearchQueryFactoryOperation subject;
+    private SearchQueryFactoryOperation subjectTest;
```

Comment:
Do you have to use 2 different subjects? If not, use the available subject.
If you need 2 different subjects, please name them both so that each name reflects what the particular object is for. In this case, even subject1 and subject2 could work better than subject and subjectTest.

Reply:
Yes two subjects are needed as the UserAuthorisation object should have AccessLevel Defined to cover the code that hits the Case_Users table. updated the changes. Please review.